eng/Versions.props

```diff
@@ -23,7 +23,7 @@
     <VersionFeature21>30</VersionFeature21>
     <VersionFeature31>32</VersionFeature31>
     <VersionFeature50>17</VersionFeature50>
-    <VersionFeature60>$([MSBuild]::Add($(VersionFeature), 13))</VersionFeature60>
+    <VersionFeature60>$([MSBuild]::Add($(VersionFeature), 12))</VersionFeature60>
```

Comment:
Why is this dropping to 12?

Reply:
In my haste to get the branding changes merged, I accidentally merged Sean's for 7.0.205 branding without modifying it to account for the hotfix first. I meant to modify that PR to go from 204 to 206 and downgrade the implicit version all in a single PR. Since that was already merged, I sent out this to increase an extra version to account for the hotfix and downgrade the implicit version at the same time.